src/js/tasks-function.js

```diff
@@ -0,0 +1,223 @@
+// eslint-disable-next-line max-classes-per-file
+class Tasks {
+  constructor(description, completed = false, index) {
+    // eslint-disable-next-line no-unused-expressions
+    (this.description = description);
+    (this.completed = completed);
+    (this.index = index);
+  }
+}
+
+class TaskStatus {
+  constructor() {
+    this.completed = false;
+  }
+
+  checked = (task) => {
+    task.completed = true;
+  };
+
+  unchecked = (task) => {
+    task.completed = false;
+  };
+}
+
+class Todolist {
+  constructor() {
+    this.taskData = [];
+  }
+
+  displayTask = () => {
+    const taskList = document.querySelector('.task-list');
+    if (taskList) {
+      taskList.innerHTML = '';
+    }
+    this.taskData.forEach((elem) => {
+      const li = [];
+      li[elem.index] = document.createElement('li');
+      li[elem.index].setAttribute('id', elem.index);
+      li[elem.index].classList.add('flex');
+      const label = [];
+      label[elem.index] = document.createElement('label');
+      label[elem.index].textContent = elem.description;
+      label[elem.index].classList.add('label');
+      label[elem.index].setAttribute('id', elem.index);
+      label[elem.index].contentEditable = true;
+      const inputBox = [];
+      inputBox[elem.index] = document.createElement('input');
+      inputBox[elem.index].setAttribute('type', 'checkbox');
+      inputBox[elem.index].setAttribute('id', elem.index);
+      inputBox[elem.index].classList.add('checkbox');
+      const moveBtn = [];
+      moveBtn[elem.index] = document.createElement('button');
+      moveBtn[elem.index].setAttribute('id', elem.index);
+      moveBtn[elem.index].innerHTML = '<i class="fa-solid fa-ellipsis-vertical ellipsis"></i>';
+
+      // const deleteBtn = [];
+      // deleteBtn[elem.index] = document.createElement('i');
+      // deleteBtn[elem.index].setAttribute('id', elem.index);
+      // deleteBtn[elem.index].classList.add('fa-regular fa-trash-can delete');
+      li[elem.index].append(
+        inputBox[elem.index],
+        label[elem.index],
+        // deleteBtn[elem.index],
+        moveBtn[elem.index],
+      );
+      taskList.append(li[elem.index]);
+      li[elem.index].addEventListener('click', () => {
+        li[elem.index].style.backgroundColor = '#ff98008f';
+
+        label[elem.index].addEventListener('click', (e) => {
+          e.target.nextSibling.innerHTML = '<i class="fa-solid fa-trash-can"></i>';
+          e.target.nextSibling.style.cursor = 'pointer';
+
+          // deleteBtn[elem.index].classList.add('active');
```

Comment:
- [ ] Kindly delete any commented code as that makes your project look unprofessional.